test/controllers/whototell.controller.test.ts

```diff
@@ -0,0 +1,59 @@
+import 'reflect-metadata'
+
+import { assert } from 'chai'
+import { BAD_REQUEST, MOVED_TEMPORARILY, OK } from 'http-status-codes'
+import request from 'supertest'
+import { deepEqual, instance, mock, when } from 'ts-mockito'
+
+import { createApp } from 'app/application.factory'
+import 'app/controllers/whototell.controller'
+import ValidationErrors from 'app/models/validationErrors'
+import { WHO_TO_TELL_URI } from 'app/paths'
+import formSchema from 'app/schemas/whototell.schema'
+import FormValidator from 'app/utils/formValidator.util'
+
+describe.only('FormController', () => {
+  describe('GET - ensure that page loads correctly', () => {
+    it('should render the form page', async () => {
+      const app = createApp()
+
+      const res = await request(app).get(WHO_TO_TELL_URI).expect(OK)
+
+      assert.include(res.text, 'Who to tell about the company closing')
+    })
+  })
+
+  describe('POST - ensure form submission is handled correctly', () => {
+    it('should redirect successfully if validator returns no errors', async () => {
+      const testObject = {test: 'data'}
+
+      const mockedFormValidator = mock(FormValidator)
+      when(mockedFormValidator.validate(deepEqual(testObject), formSchema)).thenReturn(null)
+
+      const app = createApp(container => {
+        container.rebind(FormValidator).toConstantValue(instance(mockedFormValidator))
+      })
+
+      const res = await request(app).post(WHO_TO_TELL_URI).send(testObject).expect(MOVED_TEMPORARILY)
+
+      assert.include(res.text, 'Found. Redirecting to /close-a-company/search-company')
```

Comment:
I don't think we need this. L37 provides the redirect coverage, and we can assert the redirect url with:
```
        .expect(MOVED_TEMPORARILY)
        .expect('Location', SEARCH_COMPANY_URI)
```